html-forms/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+</head>
+<body>
+
+  <form action="">
+    <div>
+      <label for="email">Email</label><br>
+      <input type="url" id="email">
+    </div>
+
+    <div>
+      <label for="choose">Why is coding cool?</label><br>
+      <select name="option choosing" id="choose">
+        <option value="option">Select a option</option>
+        <option value="interesting">Because its interesting</option>
+        <option value="fun">Because its fun</option>
+        <option value="powerful">Because its powerful</option>
+      </select>
+    </div>
+
+    <br>
+    <div>
+      <label for="Fav">Favorite Language(choose one)</label>
```

Comment:
This should just be a `p` element, it isn't labelling a single specific form control element

Reply:
Done. I also changed a couple of things after you explained a bit more.

https://user-images.githubusercontent.com/79817237/206008352-078f49b8-e89a-4f2b-a1a1-e9232b1da4d8.mp4